nav2_assisted_teleop/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 3.8)
+project(nav2_assisted_teleop)
+
+find_package(ament_cmake REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(nav_2d_utils REQUIRED)
+find_package(nav2_common REQUIRED)
+find_package(nav2_core REQUIRED)
+find_package(nav2_costmap_2d REQUIRED)
+find_package(nav2_msgs REQUIRED)
+find_package(nav2_util REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_action REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(rclcpp_lifecycle REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(tf2 REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
+
+nav2_package()
+
+include_directories(
+  include
+)
+
+add_executable(${PROJECT_NAME}
```

Comment:
Missing component Cmake / registration in the library file so that this can be loaded / used as a component, not only a standalone process server. See other packages like nav2_planner for reference of how.

Reply:
Then the library needs to be installed using macro like so: https://github.com/ros-planning/navigation2/blob/main/nav2_planner/CMakeLists.txt#L69-L73